### DISCUSSION TITLE
error upgrade 2.3.70 to 2.3.100

### DISCUSSION THREAD

**Discussion by elenezet** [Security-Onion-Solutions/securityonion#7143] [General] [answered]
When i upgrade i'he this error : 
curl: (6) Could not resolve host: raw.githubusercontent.com; Unknown error
Command failed with error 6
Could not pull signature key file, please ensure connectivity to https://raw.githubusercontent.com

Any idear?

**Comment by dougburks** [comment, marked as answer]
It looks like you've already asked this at https://github.com/Security-Onion-Solutions/securityonion/discussions/7065. Please do not create duplicate discussions.
